website/src/components/Container/index.js

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Why are these layout components being added? Are they no longer included in Docusaurus by default?

Reply:
Yeah, they have been removed from v2 so I've temporarily added them, I hope I won't need them or I'll write them differently.